packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -86,7 +86,7 @@ export interface PreferenceService extends Disposable {
     } | undefined;
 
     overridePreferenceName(options: OverridePreferenceName): string;
-    overridenPreferenceName(preferenceName: string): OverridePreferenceName | undefined;
+    overriddenPreferenceName(preferenceName: string): OverridePreferenceName | undefined;
```

Comment:
This among the other public changes would be a breaking change I mentioned in the issue to anyone implementing the interface `PreferenceService`. @akosyakov do you have a way forward not to break, or is it fine to add a note in the changelog?

Reply:
a breaking note should be fine, i doubt anyone is using it outside Theia